pdc/apps/release/models.py

```diff
@@ -327,20 +336,41 @@ def export(self):
         }
 
 
+class CPE(models.Model):
+    """
+    CPE for release variant.
```

Comment:
I am assuming CPE is common, not specific to release variant

Reply:
Oops, forgot to update the comments. Fixing it.